snooty/test_parser.py

```diff
@@ -441,29 +447,27 @@ def test_roles() -> None:
             </listItem>
             <listItem>
             <paragraph>
-            <role name="rfc"
-                  label="{'type': 'text', 'value': 'RFC-1149', 'position': {'start': {'line': 7}}}"
-                  target="1149"></role>
+            <role name="rfc" target="1149"><text>RFC-1149</text></role>
             </paragraph>
             </listItem>
             <listItem>
             <paragraph>
-            <ref_role flag="~" domain="mongodb" name="binary" target="bin.mongod"></ref_role>
+            <ref_role flag="~" domain="mongodb" name="binary" target="bin.mongod"><literal />
```

Comment:
There's a comment up at the top of the file explaining why this is empty in this file: the postprocessor isn't run to clean this up. Its emptiness indicates that it can be populated at that point.

Reply:
My bad! Makes sense 👍